config/public-api-v1.yml

```diff
@@ -31,31 +31,365 @@ paths:
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_maths:
+          type: string
+          description: "Maths GCSE requirements for applicants."
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        required_qualifications_science:
+          type: string
+          description: "Science GCSE requirements for applicants."
+          enum:
+            - ""
+            - must_have_qualification_at_application_time
+            - expect_to_achieve_before_training_begins
+            - equivalence_test
+            - not_required
+        running:
+          type: boolean
+          description: "Is the course currently running."
+          example: true
+        salary_details:
+          type: string
+          description: "Salary details about this course."
+          example: |
+            To be eligible for a place on the salaried course you must have the
+            support of a school who is willing to make contributions towards
+            your salary.
+        scholarship_amount:
+          type: integer
+          description: "The scholarship amount a candidate may be elligible for this course."
+          example: 17000
+        start_date:
+          type: string
+          format: date
+          description: "Date that the course starts."
+          example: "2020-09-01"
+        state:
+          type: string
+          description: |
+            The state of the course in the postgraduate teacher training system.
+          enum:
+            - empty
+            - rolled_over
+            - draft
+            - published
+            - published_with_unpublished_changes
+            - withdrawn
+        study_mode:
+          type: string
+          description: "Whether the course is full-time, part-time or both."
+          enum:
+            - both
+            - full_time
+            - part_time
+        summary:
+          type: string
+          description: "Generated summary of the course."
+          example: "PGCE with QTS full time"
+    CourseData:
+      type: object
+      required:
+        - id
+        - type
+        - attributes
+      properties:
+        id:
+          type: integer
+        type:
+          type: string
+        attributes:
+          $ref: "#/components/schemas/CourseAttributes"
+        relationships:
+          $ref: "#/components/schemas/CourseRelationships"
+    CourseRelationships:
+      type: object
+      properties:
+        accredited_body:
+          $ref: "#/components/schemas/Relationship"
+        provider:
+          $ref: "#/components/schemas/Relationship"
+        subjects:
+          $ref: "#/components/schemas/RelationshipList"
+    # Included:
```

Comment:
Do we need these comments?

Reply:
I'm still filling these parts in (PR is still in draft).